Synergism.css

```diff
@@ -179,12 +215,22 @@ button:focus {
     transform: translate(-50%, -50%);
     margin: 0;
     text-align: center;
+    white-space: pre-line;
 }
 
 #notifx {
-    position: fixed;
-    top: 5px;
-    right: 5px;
+    position: absolute;
```

Comment:
does this need to be absolutely positioned?

Reply:
returned because I did not know the specifications using "position: absolute;" in the child element of "position: fixed;".